vendor.conf

```diff
@@ -12,7 +12,7 @@ github.com/containernetworking/cni v0.7.0-alpha1
 github.com/containernetworking/plugins 1562a1e60ed101aacc5e08ed9dbeba8e9f3d4ec1
 github.com/containers/image 134f99bed228d6297dc01d152804f6f09f185418
 github.com/containers/storage 17c7d1fee5603ccf6dd97edc14162fc1510e7e23
-github.com/containers/psgo 382fc951fe0a8aba62043862ce1a56f77524db87
+github.com/containers/psgo master
```

Comment:
I prefer referencing the commit IDs even if it's HEAD of master. In case things go south, it's easier to track down potential sources. That's not a blocker for this PR at all but rather on my wishlist.

Reply:
Rebased without "MASTER"